pkgs/rocket-league/bakkesmod.nix

```diff
@@ -0,0 +1,71 @@
+{
+  lib,
+  makeDesktopItem,
+  symlinkJoin,
+  writeShellScriptBin,
+  unzip,
+  wine,
+  pname ? "rocket-league",
+  location ? "$HOME/Games/${pname}",
+}: let
+
+  bakkesmodIcon = builtins.fetchurl {
```

Comment:
We should use `pkgs.fetchurl` as `builtins` fetchers use IFD and we want to avoid that.
(If there are other such occurrences in other derivations, I'll fix them)